### PR TITLE
Add launch name and url to header (#583)

### PR DESCRIPTION
**Pull request by baev**

### CODE REVIEW

**Comment by baev** [comment]
[Demo.](http://ci.qatools.ru/job/allure-core_pull-request/122/artifact/allure-report-preview/target/allure-report/index.html#/) Summon @allure-framework/owners @lanwen

File: allure-report-data/src/main/groovy/ru/yandex/qatools/allure/data/io/ReportWriter.groovy [hunk 1]
```diff
@@ -62,6 +69,15 @@ class ReportWriter {
 
     void write(PluginData data) {
         Objects.requireNonNull(data)
+
+        //TODO :(
```
Comment (lanwen) [inline allure-report-data/src/main/groovy/ru/yandex/qatools/allure/data/io/ReportWriter.groovy]: what it means?

File: allure-report-data/src/main/groovy/ru/yandex/qatools/allure/data/io/ReportWriter.groovy [hunk 2]
```diff
@@ -38,6 +39,12 @@ class ReportWriter {
 
     private Long size = 0;
 
+    private String name
```
Comment (lanwen) [inline allure-report-data/src/main/groovy/ru/yandex/qatools/allure/data/io/ReportWriter.groovy]: why not just save environment obj?

**Comment by lanwen** [comment]
I like this idea, but how we can use it inside of aqua?

File: allure-report-data/src/main/groovy/ru/yandex/qatools/allure/data/io/ReportWriter.groovy (hunk 1 above)
Comment (baev) [inline allure-report-data/src/main/groovy/ru/yandex/qatools/allure/data/io/ReportWriter.groovy]: That this code as shitty as possible.

File: allure-report-data/src/main/groovy/ru/yandex/qatools/allure/data/io/ReportWriter.groovy (hunk 2 above)
Comment (baev) [inline allure-report-data/src/main/groovy/ru/yandex/qatools/allure/data/io/ReportWriter.groovy]: `Environment` also contains parameters.

**Comment by baev** [comment]
> I like this idea, but how we can use it inside of aqua?

@lanwen it is already integrated.

**Comment by lanwen** [comment]
LGTM

**Pull request merged**